pkgs/applications/misc/rxvt-unicode-plugins/default.nix

```diff
@@ -0,0 +1,22 @@
+{ callPackage }:
+
+{
+  autocomplete-all-the-things = callPackage ./urxvt-autocomplete-all-the-things { };
+
+  bidi = callPackage ./urxvt-bidi { };
+
+  font-size = callPackage ./urxvt-font-size { };
+
+  perl = callPackage ./urxvt-perl { };
+
+  perls = callPackage ./urxvt-perls { };
+
+  resize-font = callPackage ./urxvt-resize-font { };
+
+  tabbedex = callPackage ./urxvt-tabbedex { };
+
+  theme-switch = callPackage ./urxvt-theme-switch { };
+
+  vtwheel = callPackage ./urxvt-vtwheel { };
+
```

Comment:
I'm seeing these used to be in `all-packages.nix` but are now available under `nixpkgs.rxvt-unicode.plugins`.
Are there aliases for these?

Reply:
No, right, I have to add aliases for these as well.

Reply:
Ok, done.